e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/SchedulingTests.java

```diff
@@ -136,42 +139,45 @@ void afterEach() throws IOException {
     hasura.deleteMissionModel(modelId);
   }
 
+  private int directiveId1 = -1;
+  private int directiveId2 = -1;
```

Comment:
I don't think these fields aren't being cleaned up between tests (Resetting them to `-1` in the `beforeEach` should resolve this)

Reply:
I've reset these in `beforeEach` as suggested